vocabs/oada/index.js

```diff
@@ -510,4 +524,7 @@ register('hour-index', {
 // End of known terms, here are helpful functions used above:
 //------------------------------------------------------------------------
 
+// Import custom isoblue definitions
+require('vocabs/oada/isoblue');
```

Comment:
Cannot import the isoblue module at the beginning of the file as it needs the `register`, `override`, and `enumschema` functions defined later in this file. Feels a bit weird and not sure if this is the best place for it

Reply:
Per in person review with Ault, is okay